src/clj_http/multipart.clj

```diff
@@ -125,6 +125,28 @@
   [{:keys [^ContentBody content]}]
   content)
 
+(defn- multipart-workaround
+  "Workaround for AsyncHttpClient to bypass 25kb restriction on getContent.
+
+  See https://github.com/dakrone/clj-http/issues/560,
+  https://issues.apache.org/jira/browse/HTTPASYNC-163
+  "
+  [^org.apache.http.entity.mime.MultipartFormEntity mp-entity]
+  (reify org.apache.http.HttpEntity
+    (isRepeatable [_] (.isRepeatable mp-entity))
+    (isChunked [_] (.isChunked mp-entity))
+    (isStreaming [_] (.isStreaming mp-entity))
+    (getContentLength [_] (.getContentLength mp-entity))
+    (getContentType [_] (.getContentType mp-entity))
+    (getContentEncoding [_] (.getContentEncoding mp-entity))
+    (consumeContent [_] (.consumeContent mp-entity))
+    (getContent [_]
+      (let [os (java.io.ByteArrayOutputStream.)]
+        (.writeTo mp-entity os)
+        (.flush os)
+        (java.io.ByteArrayInputStream. (.toByteArray os))))
```

Comment:
I've removed these lines of code: https://github.com/apache/httpcomponents-client/blob/4.5.x/httpmime/src/main/java/org/apache/http/entity/mime/MultipartFormEntity.java#L100-L104